src/util/util_errors.h

```diff
@@ -137,6 +137,7 @@ enum sssd_errors {
     ERR_SSL_FAILURE,
     ERR_UNABLE_TO_VERIFY_PEER,
     ERR_UNABLE_TO_RESOLVE_HOST,
+    ERR_UID_OUTSIDE_RANGE,
```

Comment:
Maybe ERR_UID_OUTSIDE_OF_RANGE would be a little bit more clear? (Keep in mind that I'm not a native english speaker ...)

Reply:
Same comment with s/UID/ID/ here.

Reply:
@fidencio I believe it hardly makes any difference in adding Preposition 'of' here.
Since in enums we want to convey succinct meaning. Using without preposition is what I feel we can/often do.

Changed to ERR_ID_OUTSIDE_RANGE